### DISCUSSION TITLE
Missing blocks

### DISCUSSION THREAD

**Discussion by princeje** [mhostetter/gr-adsb#47] [General]
I'm new to GNUradio and when I open adsb_rx.grc file, gnuradio-companion is showing "Missing blocks". Do I need to build something or get the gnuradio repo cloned and put this inside of it to build it?? I'm lost.

**Comment by mhostetter** [comment]
You need to install the `gr-adsb` out-of-tree module first. Follow the [installation instructions](https://github.com/mhostetter/gr-adsb#installation) in the README and then restart `gnuradio-companion`. Let me know if you continue to have issues.
